utils/network.go

```diff
@@ -109,12 +109,27 @@ func DoNetworking(args *skel.CmdArgs, conf NetConf, result *current.Result, logg
 
 			// Handle IPv6 routes
 			if addr.Version == "6" {
+				// Make sure ipv6 is enabled in the container/pod network namespace.
+				// Without these sysctls enabled, interfaces will come up but they won't get a link local IPv6 address
+				// which is required to add the default IPv6 route.
+				if err = writeProcSys("/proc/sys/net/ipv6/conf/all/disable_ipv6", "0"); err != nil {
+					return err
```

Comment:
Will this error be descriptive enough?

I think we should wrap all of these so there's more context. 

```
return fmt.Errorf("failed to set conf/all/disable_ipv6=0: %s", err)
```

Reply:
I've updated all the sysctl errors. I used the syntax used for working on sysctl commands so it's more user-friendly.